toolchain/semantics/semantics_node.h

```diff
@@ -431,6 +433,8 @@ class SemanticsNode {
 
   auto parse_node() const -> ParseTree::Node { return parse_node_; }
   auto kind() const -> SemanticsNodeKind { return kind_; }
+
+  // Get the type of the value produced by evaluating this node.
```

Comment:
```suggestion
  // Returns type of the value produced by evaluating this node.
```

> Function comments should be written with an implied subject of _This function_ and should start with the verb phrase; for example, "Opens the file", rather than "Open the file".
https://google.github.io/styleguide/cppguide.html#Function_Comments